app/src/main/res/values/colors.xml

```diff
@@ -3,13 +3,15 @@
     <!-- Brand and base colors -->
     <color name="jellyfin_primary">#202020</color>
     <color name="jellyfin_primary_dark">#101010</color>
+    <!-- Secondary -->
     <color name="jellyfin_accent">#00a4dc</color>
 
     <!-- Launcher icon background -->
     <color name="ic_launcher_background">#000b25</color>
 
     <!-- App colors -->
     <color name="theme_background">#101010</color>
+    <color name="theme_surface">#363636</color>
```

Comment:
Do we want to keep using xml files for theming stuff or hardcode them in AppTheme/constants? (I prefer the latter)

Reply:
I think we can hardcode them, yes.

Reply:
And done.